libs/designsystem/page/src/page.component.ts

```diff
@@ -568,6 +584,19 @@ export class PageComponent
     };
   }
 
+  private patchIonLastFocussed() {
```

Comment:
Consider using the more common spelling
```suggestion
  private patchIonLastFocused() {
```